src/util/fs.js

```diff
@@ -541,7 +543,11 @@ export async function copyBulk(
       const cleanup = () => delete currentlyWriting[data.dest];
       reporter.verbose(reporter.lang('verboseFileCopy', data.src, data.dest));
       return (currentlyWriting[data.dest] = readFileBuffer(data.src)
-        .then(d => {
+        .then(async d => {
+          // we need to do this because of case-insensitive filesystems, which wouldn't properly
+          // change the file name in case of a file being renamed
+          await unlink(data.dest);
```

Comment:
Great work, @arcanis.
I am a bit worried about the extra IO operation here.
What impact would it cause for normal installation?